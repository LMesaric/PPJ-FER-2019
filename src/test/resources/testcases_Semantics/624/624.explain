semantic: i already declared
note: 'i' on line 2
