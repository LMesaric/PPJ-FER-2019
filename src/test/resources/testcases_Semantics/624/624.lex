KR_INT 1 int
IDN 1 i
TOCKAZAREZ 1 ;
KR_INT 2 int
IDN 2 i
TOCKAZAREZ 2 ;
