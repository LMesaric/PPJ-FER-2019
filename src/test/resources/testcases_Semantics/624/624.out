<izravni_deklarator> ::= IDN(2,i)
